### ISSUE TITLE
trouble building

### ISSUE THREAD

**Issue by SamKacer** [thomaseding/hearthshroud#24]
Hello, I am interested in making an accessible version of HS for blind/visually impaired gamers. The regular HS GUI lacks the accessibility markup that would allow screen readers to successfully interact with it. instead I want to utilize a reverse-engineered HS engine to create an accessible UI over. For a start a text based game running in a terminal would be an MVP. this project looks very promising in helping achieve that.

I am having a hard tiem figuring out how to build it so I could test it out. By any chance do you have an executable prebuilt for windows 10 I could run to get a feel for what features there are so far?

I am familiar with Haskell in terms of writing code in it, but I do have trouble with using cabal. I just installed latest cabal-install and ghc from chocolatey:
```cabal-install version 3.2.0.0```
```The Glorious Glasgow Haskell Compilation System, version 8.10.2```

running `cabal build` fails during resolving of dependencies. attaching error log. From what I understand, the installed Base for me is higher version than what is required by the project. running `cabal build --allow-newer` also eventually fails, because cabal can't resolve some import statements. attaching the error log for that as well.

I am not sure if I am in the infamous "cabal hell" I heard about :)

would you please recommend how I could resolve these issues? do I need to uninstall my current ghc and downgrade it to a specific version?

would it be possibly worth it to convert this project to use stack, so building could be more reproducible?

[cabal_allow_newer_errors.log](https://github.com/thomaseding/hearthshroud/files/5577581/cabal_allow_newer_errors.log)
[cabal_build_error.log](https://github.com/thomaseding/hearthshroud/files/5577582/cabal_build_error.log)
